browser/components/BoardShowPage/MenuSideBar/ActivityPanel.js

```diff
@@ -2,20 +2,29 @@ import React, { Component } from 'react'
 import Link from '../../Link'
 import moment from 'moment'
 import './ActivityPanel.sass'
+import $ from 'jquery'
 
 class ActivityPanel extends Component {
 
   static PropTypes = {
     board: React.PropTypes.object.isRequired
   }
 
+  constructor(props) {
+    super(props)
+    this.state = {
+      refs: ''
```

Comment:
hmm refs is already a react thing. Seems weird to have something called refs in your state. What is this for?